docs/backend/guides/service-objects/overview.md

```diff
@@ -20,6 +20,22 @@ Our "service objects," as we call them (this is MilMove-specific terminology), a
 package that implement our business logic. An example of a service object would be something like `AddressUpdater` 
 to update an `Address` record, or `MTOShipmentCreator` to create an `MTOShipment` record. 
 
+### Where Do Service Objects Fit In?
```

Comment:
@ronaktruss, we'd talked about this in one of our calls. Does this section get at what you were asking for?